src/content/en/tools/chrome-devtools/profile/network-performance/resource-loading.markdown

```diff
@@ -385,122 +388,213 @@ The Frames tab shows messages sent or received over a WebSocket connection. This
+(![filters button](imgs/filters.png){:.inline})
+to hide or display the **Filters** pane.
+
+Use the content type buttons to only display resources of the selected 
+content type. Tip: hold <kbd>cmd</kbd> and then click on one of these
+buttons to enable multiple content types simultaneously.
+
+![multiple content type filters selected 
+simultaneously](imgs/multiple-content-type-filters.png)
+
+The **filter** text field is deceptively powerful. If you enter an
+arbitrary string in it, the **Network** panel only displays the resources whose
+filenames match the given string.
+
+![resource name filtering](imgs/resource-name-filtering.png)
+
+The **filter** text field also supports various keywords that let you 
+sort resources by various properties, such as file size using the `larger-than`
+keyword.
+
+![filtering by file size](imgs/larger-than.png)
+
+The list below describes all of the keywords. 
+
+* `domain`: Only display resources from the specified domain. You can use 
+  a wildcard character (`*`) to include multiple domains. For example, `*.com` 
+  displays resources from all domain names ending in `.com`. DevTools 
+  populates the autocomplete dropdown menu with all of the domains
+  it has encountered.
+* `has-response-header`: Show the resources that contain the specified 
+  HTTP response header. DevTools populates the autocomplete dropdown with 
+  all of the response headers that it has encountered.
+* `is`: Use `is:running` to find `WebSocket` resources.
+* `larger-than`: Show resources that are larger than the specified size, 
+  in bytes. Setting a value of `1000` is equivalent to setting a value of `1k`.
+* `method`: Show resources that were retrieved over a specified HTTP method
+  type. DevTools populates the dropdown with all of the HTTP methods it
+  has encountered.
+* `mime-type`: Show resources of a specified MIME type. DevTools populates the
+  dropdown with all MIME types it has encountered.
+* `mixed-content`: Show all mixed content resources (`mixed-content:all`) or
+  just the ones that are currently displayed (`mixed-content:displayed`).
+* `scheme`: Show resources retrieved over unprotected HTTP (`scheme:http`) 
+  or protected HTTPS (`scheme:https`).
+* `set-cookie-domain`: Show the resources that have a `Set-Cookie` header 
+  with a `Domain` attribute that matches the specified value. DevTools 
+  populates the autocomplete with all of the cookie domains that it has 
+  encountered.
+* `set-cookie-name`: Show the resources that have a `Set-Cookie` header 
+  with a name that matches the specified value. DevTools populates the 
+  autocomplete with all of the cookie names that it has encountered.
+* `set-cookie-value`: Show the resources that have a `Set-Cookie` header
+  with a value that matches the specified value. DevTools populates the 
+  autocomplete with all of the cookie values that it has encountered.
+* `status-code`: Only show resources whose HTTP status code match the 
```

Comment:
This list is not styles like other lists. It uses code formatting vs bolding for the items being defined. Should probably update this list to by styled like the others for consistency. Also verifying each definition list is styled consistently.